sp7_db_setup_check.sh

```
@@ -0,0 +1,312 @@
+set -euo pipefail
+
+# This script sets up a master user for Specify 7 to use in MySQL/MariaDB using env vars from the Linux environment.
+
+echo "Starting MariaDB database and user creation script..."
+
+# Read variables from environment
+DB_HOST="${DATABASE_HOST}"
+DB_PORT="${DATABASE_PORT}"
+MASTER_USER_NAME="${MASTER_NAME:-${MASTER_USER_NAME:-}}"
+MASTER_USER_PASSWORD="${MASTER_PASSWORD:-${MASTER_USER_PASSWORD:-}}"
+MIGRATOR_NAME="${MIGRATOR_NAME}"
```

Shellcheck (0.11.0)

[info] This variable is assigned to itself, so the assignment does nothing.

(SC2269)